src/helpers/datatypes/did_types.rs

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
Can you check, which types you can imported from `vade-sidetree` instead of being defined here? The public available types have changed and we might be able to import a lot from there.

The type `DidDocumentResult` defined here should be preferred over a similar type from `vade-sidetree` as the type here avoids unnecessary deserialization and serialization to extract the did document (and `DidDocumentResult` as defined here might be included in `vade-sidetree` in the future).

Reply:
done

Reply:
exported sidetree-client and updated types